torch_tvm/operators.cpp

```diff
@@ -117,6 +120,23 @@ uint64_t getNoneSentinel() {
   return 0xe4fa3adecabcf036;
 }
 
+tvm::relay::Expr insertDims(const tvm::relay::Expr& in, int64_t input_dims,
```

Comment:
may as well register these as ops?

Reply:
@bwasti, this is just a helper function to wrap to expand_dims.